au.org.access-nri/model/access-om2/experiment/reproducibility/checksums/1-0-0.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Access-OM2 Checksums",
+    "description": "The checksums from the model output of Access-OM2 model run",
+    "type": "object",
+    "properties": {
+        "schema": {
```

Comment:
It seems the cool kids use `$id`

https://json-schema.org/understanding-json-schema/basics#declaring-a-unique-identifier

https://json-schema.org/understanding-json-schema/structuring#dollarid

Reply:
We're cool, so I think we can use it too.

Reply:
Although, how would we use it? Do we just point to the future https://raw.githubusercontent.com/ACCESS-NRI/schema/main/au.org.access-nri/model/access-om2/experiment/reproducibility/checksums/1-0-0.json ?

Reply:
Why is it the future? We're not using tagging.

Reply:
Because it hasn't been merged into main at this point, so that URL wouldn't resolve correctly (yet)

Reply:
Ahhh .... good point. Hrm. I suppose so, but a bit annoying.